ykrt/src/compile/jitc_yk/trace_builder.rs

```diff
@@ -18,6 +18,10 @@ struct TraceBuilder<'a> {
     jit_mod: jit_ir::Module,
     // Maps an AOT instruction to a jit instruction via their index-based IDs.
     local_map: HashMap<aot_ir::InstructionID, jit_ir::InstrIdx>,
+    // Block containing the curent control point.
```

Comment:
typo' *current*

What do mean mean by "current control point"? The control point from which we started tracing?

Reply:
Yes. I first wrote `the control point` but wanted to be more specific as there could be multiple ones.

Reply:
`current` may not be the correct term here though. Since I have to edit this anyway I'm open to suggestions.

Reply:
I think the name is ok, just perhaps describe what we mean in the comment?

Reply:
[71f8583](https://github.com/ykjit/yk/pull/998/commits/71f85836766cce02c158542583af4a0fa24ef398)